main.go

```diff
@@ -102,7 +116,7 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 		s.ChannelMessageSend(m.ChannelID, "pong!")
 	}
 
-	if match, _ := regexp.MatchString("![a-zA-Z]+ [a-zA-Z]+", m.Content); match {
+	if match, _ := regexp.MatchString("![a-zA-Z]+ [a-zA-Z\"]+ [0-9/]*", m.Content); match {
```

Comment:
delegating command matching to another package looking pretty useful right about now

Reply:
same as above

Reply:
https://github.com/BryanSLam/discord-bot/issues/19